bin/check_files.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
Is this part of the smoke test? It might be worth making this obvious in some way if so.